prisma/seed.ts

```diff
@@ -181,6 +181,7 @@ async function seedNotionMetadata() {
 
 async function seedVideoMetadata() {
   try {
+    console.log('seeding video meta data');
```

Comment:
remove

Reply:
on it

Reply:
done

Reply:
there are several other consoles which are not added by me, I can remove them as well if you want.